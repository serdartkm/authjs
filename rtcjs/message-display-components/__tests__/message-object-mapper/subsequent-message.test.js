import React from 'react'
import  SubsequentMessage  from '../../lib/message-object-mapper/subsequent-message'
import renderer from 'react-test-renderer'
describe("SubsequentMessageView snapshot", () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(<SubsequentMessage message="hello from subsequent message" datetime={1569393960355} />).toJSON()

        expect(tree).toMatchSnapshot();
    })
})



