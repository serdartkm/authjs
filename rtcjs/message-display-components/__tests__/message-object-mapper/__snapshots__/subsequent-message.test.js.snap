// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubsequentMessageView snapshot renders correctly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "marginLeft": 45,
    }
  }
>
  <div
    style={
      Object {
        "backgroundColor": "#FFECB3",
        "borderColor": "#9E9E9E",
        "borderRadius": 15,
        "borderStyle": "solid",
        "borderWidth": 2,
        "margin": 2,
        "maxWidth": "100%",
        "minWidth": "30%",
        "padding": 5,
        "wordBreak": "break-all",
        "wordWrap": "break-word",
      }
    }
  >
    <div>
      hello from subsequent message
    </div>
    <div
      style={
        Object {
          "fontSize": 10,
          "paddingTop": 2,
          "textAlign": "end",
        }
      }
    >
      <i
        style={
          Object {
            "backgroundColor": "#efebe9",
          }
        }
      >
        11:46:00 AM
      </i>
    </div>
  </div>
</div>
`;
